js/kuna.js

```diff
@@ -191,4 +192,11 @@ module.exports = class kuna extends acx {
         const response = await this.privateGetTradesMy (this.extend (request, params));
         return this.parseTrades (response, market, since, limit);
     }
+
+    async fetchOHLCV (symbol, timeframe = '1m', since = undefined, limits = undefined, params = {}) {
+        await this.loadMarkets ();
+        const trades = await this.fetchTrades (symbol, since, limits, params);
+        const ohlcvc = this.buildOHLCVC (trades, timeframe, since, limits);
+        return ohlcvc.map (c => c.slice (0, -1));
```

Comment:
we can't use `map`, though, because it requires method closures that are not supported in other languages with a similar syntax.